providers/class.two-factor-totp.php

```diff
@@ -74,23 +74,24 @@ public function user_two_factor_options( $user ) {
 		$this->admin_notices();
 		?>
 		<br />
-		<a href="javascript:;" onclick="jQuery('#two-factor-totp-options').toggle();"><?php esc_html_e( 'View Options &rarr;', 'two-factor' ); ?></a>
-		<div id="two-factor-totp-options" style="display:none;">
-			<?php if ( empty( $key ) ) {
-				$key = $this->generate_key();
-				$site_name = get_bloginfo( 'name', 'display' );
-				?>
-				<img src="<?php echo esc_url( $this->get_google_qr_code( $site_name . ':' . $user->user_login, $key, $site_name ) ); ?>" id="two-factor-totp-qrcode" />
-				<p><strong><?php echo esc_html( $key ); ?></strong></p>
-				<p><?php esc_html_e( 'Please scan the QR code or manually enter the key, then enter an authentication code from your app in order to complete setup', 'two-factor' ); ?></p>
-				<p>
-					<label for="two-factor-totp-authcode"><?php esc_html_e( 'Authentication Code:', 'two-factor' ); ?></label>
-					<input type="hidden" name="two-factor-totp-key" value="<?php echo esc_attr( $key ) ?>" />
-					<input type="tel" name="two-factor-totp-authcode" id="two-factor-totp-authcode" class="input" value="" size="20" pattern="[0-9]*" />
-				</p>
-			<?php } else { ?>
-				<p class="success"><?php esc_html_e( 'Enabled', 'two-factor' ); ?></p>
-			<?php } ?>
+		<?php if ( false && empty( $key ) ):
+			$key = $this->generate_key();
+			$site_name = get_bloginfo( 'name', 'display' );
+			?>
+			<a href="javascript:;" onclick="jQuery('#two-factor-totp-options').toggle();"><?php esc_html_e( 'View Options &rarr;', 'two-factor' ); ?></a>
+			<div id="two-factor-totp-options" style="display:none;">
+					<img src="<?php echo esc_url( $this->get_google_qr_code( $site_name . ':' . $user->user_login, $key, $site_name ) ); ?>" id="two-factor-totp-qrcode" />
+					<p><strong><?php echo esc_html( $key ); ?></strong></p>
+					<p><?php esc_html_e( 'Please scan the QR code or manually enter the key, then enter an authentication code from your app in order to complete setup', 'two-factor' ); ?></p>
+					<p>
+						<label for="two-factor-totp-authcode"><?php esc_html_e( 'Authentication Code:', 'two-factor' ); ?></label>
+						<input type="hidden" name="two-factor-totp-key" value="<?php echo esc_attr( $key ) ?>" />
+						<input type="tel" name="two-factor-totp-authcode" id="two-factor-totp-authcode" class="input" value="" size="20" pattern="[0-9]*" />
+					</p>
+			</div>
+		<?php else: ?>
+			<p class="success"><?php esc_html_e( 'Enabled', 'two-factor' ); ?></p>
```

Comment:
@joshbetz This "success" message is currently inside the `empty( $key )` conditional. It should be visible when the user has actually configured TOTP.